source/util.js

```diff
@@ -61,3 +64,23 @@ exports.getTagVersionPrefix = pMemoize(async options => {
 		return 'v';
 	}
 });
+
+exports.checkIgnoreStrategy = ({files}) => {
+	const npmignoreExists = existsSync(path.resolve(getRootDir(), '.npmignore'));
+
+	if (!files && !npmignoreExists) {
+		console.log(
+			`\n
+                 ⚠️  Warning ⚠️                      
```

Comment:
I'd prefer it if we could remove the padding/margin and such, and simply log out the error as a one-liner.

Reply:
updated